server/docs/authors-haven-api.yml

```diff
@@ -364,6 +364,32 @@ paths:
           description: Success. Email verification was successful
         500:
           description: Internal server errorcomponents     
+  /api/v1/users/profile:
+    get:
+      summary: Route for user to view their profile
+      description: Allow user to view their profile
+      security:
+        - Authorization: []
+      responses:
+        200:
+          description: user fetched successfully
+        404:
+          description: user does not exist
+        500:
+          description: Internal server error
+  /api/v1/users/profile/2:
```

Comment:
Is the profiles route mounted on `/users/profiles`? From your routes, it is accessed via `/profiles`. The route in this documentation does not exist

Reply:
Thanks for the feedback but this PR is still in progress